### PR TITLE
Update network-exchange-clc-managed-hosting-endpoint-guide.md

### PR DESCRIPTION
**Pull request by marcopao**
Added bullet item for migrating existing CLC accounts to dedicated VR, deleted SLA reference, added links for Availability Matrix in two spots and added reference and link to NetX Architecture Guide.

### CODE REVIEW

File: Network/network-exchange-clc-managed-hosting-endpoint-guide.md [hunk 1]
```diff
@@ -11,18 +11,22 @@
...
 * The Network Exchange fabric leverages the economics of shared networking while logically isolating network traffic between Exchanges, even within the same End User account. Dedicated access to CenturyLink Cloud is not supported with Network Exchange.
-* Currently, an End User may only add the Managed Hosting endpoint(s) present in the same metropolitan area as the serving instance of Network Exchange. Where more than one Managed Hosting endpoint is served, all may be included in a given Exchange. See the Network Exchange Availability Matrix and Configuration Guide for more information.
+* Currently, an End User may only add the Managed Hosting endpoint(s) present in the same metropolitan area as the serving instance of Network Exchange. Where more than one Managed Hosting endpoint is served, all may be included in a given Exchange. See the [Network Exchange Availability Matrix and Configuration Guide](https://www.ctl.io/knowledge-base/network/network-exchange-connectivity-matrix-configuration-guide/) for available endpoints per metro area.
```
Comment (jim-phillips) [inline Network/network-exchange-clc-managed-hosting-endpoint-guide.md]: this needs to be a reference to the kb .md article

File: Network/network-exchange-clc-managed-hosting-endpoint-guide.md [hunk 2]
```diff
@@ -11,18 +11,22 @@
...
 * An order for two instances of HAN VLAN Access must be placed and the order completed before provisioning the Managed Hosting endpoint.
+* For additional information regarding the Network Exchange architecture and use cases, please consult the [Network Exchange Architecture and Technical Guide](https://cloudhelp.ctl.io/knowledge-base/network-exchange-technical-guide/)
```
Comment (jim-phillips) [inline Network/network-exchange-clc-managed-hosting-endpoint-guide.md]: this needs to be a reference to the kb .md article

File: Network/network-exchange-clc-managed-hosting-endpoint-guide.md [hunk 3]
```diff
@@ -11,18 +11,22 @@
...
-* The desired Managed Hosting site must be supported within Network Exchange. See the Network Exchange Availability Matrix and Configuration Guide for supported data centers.
+* For any existing CLC accounts that wish to use Network Exchange, it is necessary to migrate them to a Dedicated VR.  Having a dedicated virtual routing table allows each customer to have their own set of IPs’ from a cloud connectivity standpoint.  A Dedicated VR is also required for automation to work when creating connectivity to the cloud.  
+To create a dedicated virtual routing table (Dedicated VR), the VR will need to be created and all existing IP blocks the customer is using will need to converge to the new VR.  To initiate this process, create a ticket with the CenturyLink Cloud help desk.  
+
+* The desired Managed Hosting site must be supported within Network Exchange. See the [Network Exchange Availability Matrix and Configuration Guide](https://www.ctl.io/knowledge-base/network/network-exchange-connectivity-matrix-configuration-guide/) for supported data centers.
```
Comment (jim-phillips) [inline Network/network-exchange-clc-managed-hosting-endpoint-guide.md]: this needs to be a reference to the kb .md article

**Comment by jim-phillips** [comment]
@marcopao please address comments

**Comment by jim-phillips** [comment]
@marcopao last chance to fix this.  I will be closing due to inactivity on 4/11

File: Network/network-exchange-clc-managed-hosting-endpoint-guide.md (hunk 3 above)
Comment (JasonEHolland) [reply Network/network-exchange-clc-managed-hosting-endpoint-guide.md]: Ignore this request to merge. I just updated this KB.

File: Network/network-exchange-clc-managed-hosting-endpoint-guide.md (hunk 1 above)
Comment (JasonEHolland) [reply Network/network-exchange-clc-managed-hosting-endpoint-guide.md]: Ignore this request to merge. I just updated this KB.

File: Network/network-exchange-clc-managed-hosting-endpoint-guide.md (hunk 2 above)
Comment (JasonEHolland) [reply Network/network-exchange-clc-managed-hosting-endpoint-guide.md]: Ignore this request to merge. I just updated this KB.

**Pull request closed without merging**